sdk/storage/storage-blob/test/node/emulator-tests.spec.ts

```diff
@@ -0,0 +1,133 @@
+import * as assert from "assert";
+import * as dotenv from "dotenv";
+import {
+  BlobClient,
+  ContainerClient,
+  AppendBlobClient,
+  BlobServiceClient,
+  BlockBlobClient,
+  PageBlobClient
+} from "../../src";
+import {
+  getBSU,
+  getConnectionStringFromEnvironment,
+  env,
+  bodyToString,
+  getUniqueName
+} from "../utils";
+dotenv.config({ path: "../.env" });
+
+// Expected environment variable to run this test-suite
+// STORAGE_CONNECTION_STRING=UseDevelopmentStorage=true
+describe("Emulator Tests", () => {
+  const blobServiceClient = getBSU();
+  let containerName: string;
+  let blobName: string;
+  let containerClient: ContainerClient;
+  let blobClient: BlobClient;
+  let blockBlobClient: BlockBlobClient;
+  const content = "Hello World";
+
+  beforeEach(async function() {
+    if (!env.STORAGE_CONNECTION_STRING.startsWith("UseDevelopmentStorage=true")) {
+      this.skip();
+    }
+    containerName = getUniqueName("container");
+    blobName = getUniqueName("blob");
+    containerClient = blobServiceClient.getContainerClient(containerName);
+    await containerClient.create();
+    blobClient = containerClient.getBlobClient(blobName);
+    blockBlobClient = blobClient.getBlockBlobClient();
+    await blockBlobClient.upload(content, content.length);
+  });
+
+  afterEach(async function() {
+    await containerClient.delete();
+  });
+
+  it("BlobClient can be created with a connection string", async () => {
+    const newClient = new BlobClient(getConnectionStringFromEnvironment(), containerName, blobName);
+    const metadata = {
+      a: "a",
+      b: "b"
+    };
+    await newClient.setMetadata(metadata);
+    const result = await newClient.getProperties();
+    assert.deepStrictEqual(result.metadata, metadata);
+  });
+
+  it("AppendBlobClient can be created with a connection string", async () => {
```

Comment:
![image](https://user-images.githubusercontent.com/10452642/65481335-c45e3d00-de49-11e9-868a-049d558ed335.png)
Removing this test since the feature is not supported yet.